examples/contrived.rs

```diff
@@ -0,0 +1,24 @@
+use instant_segment::{Search, Segmenter};
+use std::collections::HashMap;
+
+fn main() {
+    let mut unigrams = HashMap::default();
+
+    unigrams.insert("choose".into(), 50 as f64);
```

Comment:
I think you probably don't need all the `as` casts as long as you put a .0 after the numbers?